tests/Unit/Execution/Arguments/TypedArgsTest.php

```diff
@@ -49,6 +38,42 @@ public function testNullableList(): void
         }
         ';
 
+        $argumentSet = $this->rootQueryArgumentSet([
+            'bar' => null,
+        ]);
+
+        $this->assertCount(1, $argumentSet->arguments);
+
+        $bar = $argumentSet->arguments['bar'];
+        $this->assertInstanceOf(Argument::class, $bar);
+        $this->assertSame(null, $bar->value);
+    }
+
+    public function testNullableInputObject(): void
+    {
+        $this->schema = '
+        type Query {
+            foo(bar: Bar): Int
+        }
+        
+        input Bar {
+            baz: ID
+        }
+        ';
+
+        $argumentSet = $this->rootQueryArgumentSet([
+            'bar' => null,
+        ]);
+
+        $this->assertCount(1, $argumentSet->arguments);
+
+        $bar = $argumentSet->arguments['bar'];
+        $this->assertInstanceOf(Argument::class, $bar);
+        $this->assertSame(null, $bar->value);
```

Comment:
It is better to use specific assertions instead of generics. Try with [assertNull](https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.assertNull)